source/Playnite.DesktopApp/Controls/NumericDoubleBox.cs

```diff
@@ -1,18 +1,10 @@
 ﻿using Playnite.Common;
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Globalization;
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Data;
-using System.Windows.Documents;
 using System.Windows.Input;
-using System.Windows.Media;
-using System.Windows.Media.Imaging;
-using System.Windows.Navigation;

```

Comment:
Please don't remove usings unnecessarily unless it's a required change related to the fix.

Reply:
No problem. My VS is setup to automatically clean them, but I'll keep it in mind for the future.